src/pages/Login/Login.tsx

```diff
@@ -33,7 +36,7 @@ const Login: React.FC = () => {
         history.push("/authenticate");
       }
     } catch (e) {
-      setError(e);
+      setError(e as ApiError | NetworkError | BaseError);
```

Comment:
```suggestion
      setError(e as BaseError);
```

since `ApiError` and `NetworkError` _are_ BaseErrors, you can just do that ☝️